src/resolvers/mutations/publicCloud/requestDecision.ts

```diff
@@ -109,7 +109,7 @@ const publicCloudRequestDecision = async (
       request.requestedProject?.secondaryTechnicalLead,
     ].filter(Boolean);
 
-    Promise.all(users.map((user) => subscribeUserToMessages(user.email)));
+    Promise.all(users.map((user) => subscribeUserToMessages(user.email, "AWS", "Public")));
```

Comment:
I believe we want to pass `user` instead of `user.email` in here as well like others above?